installer/deb/create_deb.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# This script generates a .deb file for the installation of MassaStation on a Debian-based Linux distribution.
+
+set -e
+
+BUILD_DIR=builddeb
+PKGVERSION=0.0.0-dev
+
+SERVER_BINARY_NAME=massastation-server
+APP_BINARY_NAME=massastation-app
+
+# Print error message to stderr and exit with code 1.
+fatal() {
+    echo "FATAL: $1"
+    exit 1
+}
+
+# Download the latest release of MassaStation app.
+download_massastation_app() {
+    curl -L https://github.com/massalabs/Thyra-Menu-Bar-App/releases/latest/download/ThyraApp_linux-amd64 -o $APP_BINARY_NAME || fatal "failed to download $APP_BINARY_NAME"
+    chmod +x $APP_BINARY_NAME || fatal "failed to chmod $APP_BINARY_NAME"
+}
+
+# Build the MassaStation server binary.
+build_massastation_server() {
+    go generate ../... || fatal "go generate failed for $SERVER_BINARY_NAME"
+    go build -o $SERVER_BINARY_NAME ../cmd/thyra-server/ || fatal "failed to build $SERVER_BINARY_NAME"
+    chmod +x $SERVER_BINARY_NAME || fatal "failed to chmod $SERVER_BINARY_NAME"
+}
+
+# Delete the build directory if it exists.
+clean() {
+    if [ -d $BUILD_DIR ]; then
+        rm -rf $BUILD_DIR || fatal "failed to delete $BUILD_DIR"
+    fi
+}
+
+# Install dependencies required to build the .deb file.
+install_dependencies() {
+    sudo apt-get install dpkg-dev || fatal "failed to install dpkg-dev"
+}
+
+main() {
+    clean
+
+    install_dependencies
+
+    test -f $SERVER_BINARY_NAME || build_massastation_server
+    test -f $APP_BINARY_NAME || download_massastation_app
```

Comment:
why this? it is possible that the binaries already exists? and if yes do you really want use them instead of building fresh ones?

Reply:
I already answered to that question in [this comment](https://github.com/massalabs/thyra/pull/668#discussion_r1196188948)